File: benchmark/results/python-invocation-script/run-002/b-baseline/output/metaengine_demo/inventory/value_objects/warehouse_location.py
"""WarehouseLocation value object."""
from dataclasses import dataclass
from datetime import datetime


@dataclass(frozen=True)
class WarehouseLocation:
    """Immutable WarehouseLocation value object."""

    id: str
    created_at: datetime
    updated_at: datetime
    name: str
    description: str

File: benchmark/results/java-invocation-script/run-002/a-mcp/output/inventory/value_objects/WarehouseLocation.java
package com.metaengine.demo.inventory.value_objects;

import java.time.OffsetDateTime;

// WarehouseLocation value object.

public class WarehouseLocation {
    public String id;
    public OffsetDateTime createdAt;
    public OffsetDateTime updatedAt;
    public String name;
    public String description;

    public WarehouseLocation(String id, OffsetDateTime createdAt, OffsetDateTime updatedAt, String name, String description) {
        this.id = id;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
        this.name = name;
        this.description = description;
    }
}
